src/client/java/techreborn/TechRebornClient.java

```diff
@@ -24,8 +24,10 @@
 
 package techreborn;
 
+import dev.architectury.event.events.client.ClientTickEvent;
```

Comment:
Will it work with TR only, e.g. without REI ?

Reply:
Oops, forgot to clean up unused imports